src/main/java/com/crypto/portal/cryptoportal/config/MainDBConfig.java

```diff
@@ -20,6 +20,7 @@
 @EnableTransactionManagement
 @EnableJpaRepositories(
         entityManagerFactoryRef = "entityManagerFactory",
+        transactionManagerRef = "usageTransactionManager",

```

Comment:
@wasifzaman182  why change in mainDbConfig class??

Reply:
sorry i was reading about these annotations, during this got an issue and start working on that and i was thinking that i changed.

Reply:
if this is not related to your changes, kindly remove your change and re-push again. Thanks
@wasifzaman182